protobuf/scalapb/scalapb.proto

```diff
@@ -2,7 +2,10 @@ syntax = "proto2";
 
 package scalapb;
 
+option csharp_namespace = "ScalaPb.Options";
+option go_package = "scalapb.github.io/pirum.com/cda/grpc/trades";
 option java_package = "scalapb.options";
+option php_namespace = "ScalaPb\\Options";
```

Comment:
Currently, php generator only accepts proto files with proto3 syntax, so this line will not fix support for php users either. If php support is needed, let's open a different github issue to explore.

Reply:
Apologies, I was distracted when I read your comment.